webonary-cloud-api/lambda/browseEntries.ts

```diff
@@ -1,3 +1,23 @@
+/**
+ * @api {get} /browse/entry/:dictionaryId Get dictionary entries for a letter head.
```

Comment:
@johntseng, I'm 99% certain that the correct term is "Head Letter" rather than "Letter Head". You may want to ask Steve or Verna. I found this by Googling: https://patents.google.com/patent/US4541069A/en

Reply:
Good catch. Renamed in https://github.com/sillsdev/webonary/pull/313/commits/312c81fd9f9dec10c58d1064bf679d4267642d9e